src/pkg/repository/repository.go

```diff
@@ -225,7 +227,9 @@ func Connect(
 		return nil, clues.Wrap(err, "constructing event bus")
 	}
 
-	rm := &repositoryModel{}
+	rm := &repositoryModel{
+		Version: version.Repo,
```

Comment:
Initializing this to `version.Repo` in case we don't have a persisted model (see section below)